realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpNetworkTransport.java

```diff
@@ -37,14 +40,25 @@ public Response sendRequest(String method, String url, long timeoutMs, Map<Strin
             OkHttpClient client = getClient(timeoutMs);
             okhttp3.Response response = null;
             try {
-                Request.Builder builder = new Request.Builder().url(url);
-                switch(method) {
-                    case "get": builder.get(); break;
-                    case "delete": builder.delete(RequestBody.create(JSON, body)); break;
-                    case "patch": builder.patch(RequestBody.create(JSON, body)); break;
-                    case "post": builder.post(RequestBody.create(JSON, body)); break;
-                    case "put": builder.put(RequestBody.create(JSON, body)); break;
-                    default: throw new IllegalArgumentException("Unknown method type: "+ method);
+                okhttp3.Request.Builder builder = new okhttp3.Request.Builder().url(url);
+                switch (method) {
+                    case "get":
+                        builder.get();
+                        break;
+                    case "delete":
+                        builder.delete(RequestBody.create(JSON, body));
+                        break;
+                    case "patch":
```

Comment:
Nitpick: I would extract these to constants, also below

Reply:
I am going to extract the whole make a request functionality